huaweicloud/resource_huaweicloud_dns_recordset_v2.go

```diff
@@ -188,16 +188,15 @@ func resourceDNSRecordSetV2Read(d *schema.ResourceData, meta interface{}) error
 	// save tags
 	resourceType, err := getDNSRecordSetTagType(zoneType)
 	if err != nil {
-		return fmt.Errorf("Error getting resource type of DNS record set %s: %s", recordsetID, err)
+		log.Printf("[WARN] Error getting tags tpye of DNS record %s: %s", recordsetID, err)
```

Comment:
please revert this change, if we get the wrong zone type, the following statement will exec with errors.